test/Listener/LogTest.php

```diff
@@ -1,24 +1,26 @@
 <?php
 namespace Eris\Listener;
 
-use Eris\Generator\GeneratedValueSingle;
-
 class LogTest extends \PHPUnit_Framework_TestCase
 {
+    private $timezone;
     protected function setUp()
     {
-        $this->file = '/tmp/eris-log-unit-test.log';
+        $this->timezone = date_default_timezone_get();
+        $this->file = sys_get_temp_dir().'/eris-log-unit-test.log';
         $this->time = function () {
             return 1300000000;
         };
         $this->log = new Log($this->file, $this->time, 1234);
+        date_default_timezone_set('UTC');
     }
 
     public function tearDown()
     {
         $this->log->endPropertyVerification(null, null);
+        date_default_timezone_set($this->timezone);
```

Comment:
if the intent is to restore this, go for `$this->originalTimezone`. Although for the scope of this test suite forcing it to `UTC` and forgetting about it may be enough